ajax/libs/amalthea/1.1.4/amalthea.08fd947.bundle.js

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
Using `amalthea.bundle.js` as the file name is a good way let developers add it.
What do you think?
Many thanks.